internal/app/machined/pkg/controllers/kubeaccess/talossa/crd_controller.go

```diff
@@ -0,0 +1,633 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package talossa
```

Comment:
Note to reviewer: This is basically the same thing as https://github.com/kubernetes/sample-controller/blob/master/controller.go , except this one uses dynamic k8s client & unstructured to avoid code generation for the CRD.